### PR TITLE
Android speed limit view

### LINKED ISSUE #137: Android - Add Speed Limit Views
Add the MUTCD & ViennaConvention speed limit views as seen in https://github.com/stadiamaps/ferrostar/tree/main/apple/Sources/FerrostarSwiftUI/Views/SpeedLimit

### LINKED ISSUE #317: Android Speed Limit Sign
See:

- The speed limit views - https://github.com/stadiamaps/ferrostar/tree/main/apple/Sources/FerrostarSwiftUI/Views/SpeedLimit
- Snapshot tests - https://github.com/stadiamaps/ferrostar/tree/main/apple/Tests/FerrostarSwiftUITests/Views/__Snapshots__/SpeedLimitViewTests

Note, this one of the few views that must remain the same color regardless of light/dark mode.

### PR DESCRIPTION
**Pull request by Archdoog**
- Adds android Vienna & USStyle Speed Limit Views.
- Adjusts current annotation to pass wrapper through to view model ui state. 

### Demo Videos
- **US Demo** - https://vimeo.com/1039765198
- **Vienna Demo** - https://vimeo.com/1039765454

Closes #137
Closes #317

### CODE REVIEW

File: android/composeui/src/main/java/com/stadiamaps/ferrostar/composeui/views/components/speedlimit/SpeedLimitView.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+enum class SignageStyle {
+  MUTCD,
+  ViennaConvention
+}
+
+@Composable
+fun SpeedLimitView(
+    modifier: Modifier = Modifier,
+    speedLimit: MeasurementSpeed,
+    signageStyle: SignageStyle = SignageStyle.ViennaConvention, // TODO: This could be nicer
```
Comment (ianthetechie) [inline android/composeui/src/main/java/com/stadiamaps/ferrostar/composeui/views/components/speedlimit/SpeedLimitView.kt:21]: Let's remove the default (recursively); discussed on the call.

(Same on iOS)

File: android/core/src/main/java/com/stadiamaps/ferrostar/core/annotation/AnnotationWrapper.kt [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
-import com.stadiamaps.ferrostar.core.NavigationState
+import com.stadiamaps.ferrostar.core.measurement.MeasurementSpeed
+import com.stadiamaps.ferrostar.core.measurement.MeasurementSpeedUnit
 
 data class AnnotationWrapper<T>(
     val annotation: T? = null,
     val speed: Speed? = null,
-    val state: NavigationState
-)
+    //    val state: NavigationState
```
Comment (ianthetechie) [inline android/core/src/main/java/com/stadiamaps/ferrostar/core/annotation/AnnotationWrapper.kt:9]: Delete me.

File: android/demo-app/src/main/java/com/stadiamaps/ferrostar/DemoNavigationViewModel.kt [hunk 3]
```diff
@@ -19,56 +19,85 @@ import kotlinx.coroutines.flow.map
...
+  //            }
+  //          }
+  //          .stateIn(
+  //              scope = viewModelScope,
+  //              started = SharingStarted.WhileSubscribed(),
+  //              // TODO: Heading
+  //              initialValue =
+  //                  NavigationUiState(
+  //                      null, null, null, null, null, null, null, false, null, null, null, null,
+  // null))
```
Comment (ianthetechie) [inline android/demo-app/src/main/java/com/stadiamaps/ferrostar/DemoNavigationViewModel.kt:100]: Delete me

Comment (Archdoog) [reply android/demo-app/src/main/java/com/stadiamaps/ferrostar/DemoNavigationViewModel.kt]: Sorted 👍

File: android/core/src/main/java/com/stadiamaps/ferrostar/core/annotation/AnnotationWrapper.kt (hunk 2 above)
Comment (Archdoog) [reply android/core/src/main/java/com/stadiamaps/ferrostar/core/annotation/AnnotationWrapper.kt]: Sorted 👍

File: android/composeui/src/main/java/com/stadiamaps/ferrostar/composeui/views/components/speedlimit/SpeedLimitView.kt (hunk 1 above)
Comment (Archdoog) [reply android/composeui/src/main/java/com/stadiamaps/ferrostar/composeui/views/components/speedlimit/SpeedLimitView.kt]: Sorted 👍

**Review by ianthetechie — APPROVED** [review]

**Pull request merged**